sensu/server/_handler_pagerduty.sls

```diff
@@ -0,0 +1,42 @@
+
+gem_sensu_pagerduty:
```

Comment:
Hmm, now I noticed this is going to be installed into rbenv used only by sensu so it's not as big evil as I supposed at the first glance.
So maybe it's ok to merge this and then make a cleanup after when things will settle.